### PR TITLE
LUN-300: Adjustments for new HomeScreen

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

In the ticket I noted for every point if I did it / if it was already done/ or if I won't do it. With this ticket the home screen is ready for deployment, so feel free to test everything, all should work now.

### CODE REVIEW

**Comment by SteffiStuffel** [comment]
> This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail). You can find it there by looking for an issue with the key which is mentioned in the title of this pull request. It starts with the keyword _LUN_.
> 
> In the ticket I noted for every point if I did it / if it was already done/ or if I won't do it. With this ticket the home screen is ready for deployment, so feel free to test everything, all should work now.

**Review by f1sh1918 — APPROVED** [review]
nice cleanup!

**Review by steffenkleinle — APPROVED** [review]
Can you explain why `can professionSelectionScreen and disicplineSelectionScreen be merged or somehow share more stuff?` is `won't do`? Too much effort, not enough similarities, separate ticket?

File: src/routes/home/components/__tests__/NextExerciseCard.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        thumbnail='thumbnail-uri'
+        heading='heading'
+        subheading='subheading'
+        buttonLabel='button'
+        onPress={onPress}
+      />
+    )
+    expect(getByText('heading')).toBeDefined()
+    expect(getByText('subheading')).toBeDefined()
+    const image = await findByRole('image')
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/routes/home/components/__tests__/NextExerciseCard.spec.tsx]: Double standards :D 
Changed it to test-id.

**Pull request merged**